### ISSUE TITLE
problem with vcs-import command

### ISSUE THREAD

**Issue by ashishrao7** [cedric-scheerlinck/dvs_image_reconstruction#2]
Hey,

This is regarding the installation. Everything works fine until this command:
`vcs-import < dvs_image_reconstruction/dependencies.yaml`

The error is as follows:

![image](https://user-images.githubusercontent.com/12975862/48766077-9f54e700-ecd9-11e8-8395-7aa5317d8ea1.png)

The above command is being executed on Ubuntu  16.04.4 LTS with ROS kinetic installed.
I initially thought that it was a problem with my ssh keys. However, I am able to push and pull normally on my system and hence assumed that wasn't the problem.

**Comment by cedric-scheerlinck** [comment]
As a temporary fix can you try cloning each repo listed in the dependencies.yaml file one by one using "git clone <repo_address>"?

The import command will just try to clone the repos the are missing into the current directory (should be src)

**Comment by ashishrao7** [comment]
Yeah. Finally, I did that. I was kinda able to setup things in the end, but
I think I had to deviate from the readme. I’ll let you know on Monday.

**Comment by ashishrao7** [comment]
Everything works fine now. The only problem I had was with the `vcs-import` command for some reason.
